app/views/builds.html

```diff
@@ -135,6 +131,51 @@
                       </span>
                     </td>
                   </tr>
+
+                  <!-- Builds that were not spawned from a build config -->
+                  <tr ng-repeat="latestBuild in buildsNoConfig">
```

Comment:
I know this was copied from above, but should use `track by (latestBuild | uid)`